docs/developer.md

```diff
@@ -61,32 +59,45 @@ Walking through this last example, we see that invoking `myst start --headless`
 The **theme server** is a React app that knows how to style data, which it pulls in from the content server.
 The user interacts with the React app, which may trigger new fetches from the content server.
 
-### Project-specific concepts
+## Project-specific concepts
 
 `mystmd` is built on top of well known tools in the JavaScript ecosystem, such as [unist](https://github.com/syntax-tree/unist) from [unified](https://unifiedjs.com/), [mdast](https://github.com/syntax-tree/mdast), and [citation.js](https://citation.js.org/) for the `myst` CLI or [React](https://reactjs.org/), [Remix](https://remix.run/), and [Tailwind CSS](https://tailwindcss.com/) for the theme server.
 
 If you are familiar with these tools, you should not find many surprises in the codebase.
 
 That said, there are a couple of concepts used _only_ in this project, that won't be familiar. These are detailed below:
 
-#### Concepts: theme server
+### Concepts: Renderers, themes, and templates
 
 In the diagram above, we saw that `mystmd` produces websites by converting a set of documents to an AST, by serving that AST via a content server, and then exposing the data to the user via a React webapp (`myst-theme`) that pulls in data from the content server.
 
 The [`myst-theme` repository](https://github.com/jupyter-book/myst-theme/) contains the default themes that ship with MyST, and is an example of a React-based MyST theme.
 The [`myst-to-react` package](https://github.com/jupyter-book/myst-theme/tree/main/packages/myst-to-react) provides a `<MyST />` component which can render MyST AST into a React tree.
 A React [context](https://react.dev/reference/react/useContext), named `ThemeContext` (defined [here in the `myst-theme` repository](https://github.com/jupyter-book/myst-theme/blob/main/packages/providers/src/theme.tsx)), is used to push state deeply into the tree, without having to pass it via props.
 
+#### Where to find renderers, themes, and templates
```

Comment:
Are these three terms defined before they're used?

Reply:
I'll define them more clearly in the "overview" section